### ISSUE TITLE
Disappearing stories

### ISSUE THREAD

**Issue by evg-zhabotinsky** [knighty/fimfiction-issues#330]
I think there's some glitch that makes stories go missing, at least from bookshelf / author listings.
An example: https://www.fimfiction.net/user/10369/Kkat/stories
It says Kkat has 2 stories published on FimFiction, yet the listing shows none:
![kkat](https://user-images.githubusercontent.com/6692701/42890943-c1bad640-8ab6-11e8-8471-f7c2013a3b13.png)
I'm pretty sure at least "Fallout: Equestria" is supposed to be there. I'm also pretty sure I added it to my "Epic" bookshelf, which now lists only 9 stories despite its counter saying 10, and FoE isn't on the list now. ([link](https://www.fimfiction.net/bookshelf/1071922/epic))
Even if the stories are no longer supposed to be there somehow (like deleted? I don't really see any good reason), the counters are wrong then.

Might be related to various counting issues reported here, like #312 for example. At least now I suspect my "Read later" bookshelf could well have such a ghost story in it, for it claims to have 1 story more than it lists for about a year now.

**Comment by dagopher** [comment]
Did you make sure your settings were set to allow you to see mature content? If not, those stories would not appear for you.

**Comment by evg-zhabotinsky** [comment]
Well, as a matter of fact, I did not. All my counters suddenly fixed themselves now. Thanks for the tip.

It still confused the heck out of me, and might not have been the only problem. I'm pretty sure I enabled that toggle a while ago and don't remember ever touching it again, but I've most definitely read some M-rated stories both before and after my "Read It Later" acquired that "ghost story", which is now gone.

Anyway, I guess now it's a dupe of #144 and can be closed.
